docs/en/packages/fast-application-utilities/readme.md

```diff
@@ -0,0 +1,33 @@
+---
+id: index
+title: FAST Application Utilities
+sidebar_label: Application Utilities
+---

```

Comment:
I believe @chrisdholt is deprecating this package - I don't think we should document it here.